chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -254,6 +258,17 @@
 <!ENTITY zotero.proxy.recognized.disable.label			"Do not automatically redirect requests through previously recognized proxies">
 <!ENTITY zotero.proxy.recognized.ignore.label			"Ignore">
 
+<!ENTITY zotero.feedSettings.title                      "Feed Settings">
+<!ENTITY zotero.feedSettings.saveButton.label           "Save">
+<!ENTITY zotero.feedSettings.url.label                  "URL">
+<!ENTITY zotero.feedSettings.title.label                "Title">
+<!ENTITY zotero.feedSettings.refresh.label1             "Refresh Interval:">
+<!ENTITY zotero.feedSettings.refresh.label2             "hour(s)">
+<!ENTITY zotero.feedSettings.title.label                "Title">
+<!ENTITY zotero.feedSettings.cleanAfter.label1         "Remove read articles after ">
+<!ENTITY zotero.feedSettings.cleanAfter.label2         "day(s)">
```

Comment:
We do this currently in the General pane: "Automatically remove items in the trash deleted more than" [x] "days ago"

I think it's OK in theory, since localizers can put whatever they want in either side to make it make sense. If they translate literally on Transifex it might not work, but it can be fixed by someone who tries in the software itself.

Mozilla (who's generally pretty careful about its localization — and has whole pluralization systems that are far more detailed and correct than what we do) does this in a number of places in the Thunderbird prefs:

"After displaying for" [x] "seconds"
"Let my contacts know what I am idle after" [x] "minutes of activity"
"Auto Save every" [x] "minutes"

So at least to the extent that TB still represents Mozilla's practices, it seems to be OK by their standards.